src/components/AccountModal.vue

```diff
@@ -45,6 +45,27 @@ export default {
         {name: 'composition 4'}
       ]
     }
+  },
+  // declare all methods here
+  methods: {
+    logMe () {
+      console.log('Test Log from the methods declaration')
```

Comment:
is this just a place holder?

Reply:
For now, yeah. It's where I'm gonna add the logic for the "change username and password" functionality when I create my next branch,